src/test/run-make-fulldeps/coverage-reports/Makefile

```diff
@@ -1,6 +1,6 @@
 # needs-profiler-support
 # ignore-windows-gnu
-# min-llvm-version: 11.0
+# min-llvm-version: 12.0
```

Comment:
Can you keep the min at 11.0?

The only reason we added the minimum version of 11 here was because that was the first first of LLVM that introduced LLVM Coverage Mapping Version 4.

Since you are not updating the coverage mapping version, the `llvm-profdata` and `llvm-cov` tools should still work from version 11.

(See #79365 for more info.)